cypress/support/commands.js

```diff
@@ -0,0 +1,22 @@
+// ***********************************************
+// generate date command Format : Tuesday, 31 January
+Cypress.Commands.add('getDate', () => {
```

Comment:
@Rabab-Diab inside this command, instead of accessing the environment variable, the values should be passed as arguments to the method and used accordingly.
And please try not to define local variables.
If I were you, I would try to do it one statement, or two at most ;)

Reply:
This method will return the date and will be used in the test case block .. what do you think?